sublime_linter.py

```diff
@@ -251,8 +255,9 @@ def highlight(self, view, errors, hit_time):
             vid = view.id()
             persist.errors[vid] = errors_by_line
 
-        events.broadcast(events.FINISHED_LINTING, {'buffer_id': view.buffer_id()})
+        events.broadcast(events.FINISHED_LINTING, {'buffer_id': bid})
 
+        # TODO: These should go to their own modules.
```

Comment:
I'd prefer it if the TODO was done, or if it were an issue so we can track it.

Reply:
Then I open an issue.